pkgs/development/tools/build-managers/rake/gemset.nix

```diff
@@ -4,9 +4,9 @@
     platforms = [];
     source = {
       remotes = ["https://rubygems.org"];
-      sha256 = "15whn7p9nrkxangbs9hh75q585yfn66lv0v2mhj6q6dl6x8bzr2w";
+      sha256 = "1ilr853hawi09626axx0mps4rkkmxcs54mapz9jnqvpnlwd3wsmy";
```

Comment:
```suggestion
      hash = "1ilr853hawi09626axx0mps4rkkmxcs54mapz9jnqvpnlwd3wsmy";
```

Reply:
This file is automatically generated by [bundix](https://github.com/nix-community/bundix), and currently it only generated sha256 hashes.